### ISSUE TITLE
Officially upport HHVM 

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpoints#344]
We support HHVM, but we still have it as part of the allowed failures on our Travis CI build. Let's move it out of that (and maybe replace it with `hhvm-dev`?).

**Comment by JDGrimes** [comment]
We can't run against `hhvm-nightly` at the moment, see https://github.com/travis-ci/travis-ci/issues/3788#issuecomment-97171678. We won't be able to run against HHVM 3.7 either, see https://github.com/facebook/hhvm/issues/5220#issuecomment-97168446. Therefore, as much as I'd like to, I don't think that this is the proper time. We'll need to wait until Travis CI has updated and can support testing against the latest and nightly versions of HHVM.

**Comment by JDGrimes** [comment]
Just an update on this: we're still waiting for Travis CI to update their builds to run on Ubuntu Trusty, since HHVM no longer supports Precise. [Travis offers a Trusty build](https://docs.travis-ci.com/user/languages/php/#hhvm-versions-on-trusty) which offers a variety of HHVM versions (which I actually hadn't considered before, really), but it doesn't yet officially support PHP.

**Comment by JDGrimes** [comment]
[PHP is now supported on Trusty](https://docs.travis-ci.com/user/trusty-ci-environment#php), but not PHP 5.2. So let's continue to wait on this.

**Comment by JDGrimes** [comment]
The docs now say:

>Note: We’re unable to build PHP 5.2 on Trusty, so trying to use it will result in a build failure when phpenv fails to compile it

So I think it may be that Travis simply doesn't plan on supporting 5.2 on Trusty at all. But we'll see. 

Related: #271

**Comment by JDGrimes** [comment]
Note that WordPress has basically dropped implicit support for HHVM (not sure if it was ever explicit), by removing it from the Travis test matrix ([WP#40548](https://core.trac.wordpress.org/ticket/40548)). So it is probably pointless for us to worry about testing against HHVM anymore. I'm going to mark this as `wontfix`.
